tracer/src/Datadog.Trace/Ci/CIVisibility.cs

```diff
@@ -134,6 +135,11 @@ public static void Initialize()
             {
                 // Extract repository name from the git url and use it as a default service name.
                 tracerSettings.ServiceNameInternal = GetServiceNameFromRepository(CIEnvironmentValues.Instance.Repository);
+                tracerSettings.GlobalTagsInternal[CommonTags.UserProvidedTestServiceTag] = "false";
```

Comment:
Just a thought, given this is added to every span, maybe you want to add it as part of the serialization directly instead?

Reply:
Good idea, I'll do that in a different optimization PR though